sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -234,43 +236,44 @@ int main() {
 
 ```
 
-# Build and Test a simple SYCL program
-
-To build simple-sycl-app put `bin` and `lib` to PATHs and run following command:
-### Linux:
-   ```bash
-   export PATH=$SYCL_HOME/build/bin:$PATH
-   export LD_LIBRARY_PATH=$SYCL_HOME/build/lib:$LD_LIBRARY_PATH
-   ```
-### Windows:
-   ```bash
-   set PATH=%SYCL_HOME%\build\bin;%PATH%
-   set LIB=%SYCL_HOME%\build\lib;%LIB%
-   ```
-
-### Linux & Windows:
-   ```bash
-   clang++ -fsycl simple-sycl-app.cpp -o simple-sycl-app.exe -lOpenCL
-   ```
+To build simple-sycl-app put `bin` and `lib` to PATHs:
+
+**Linux**
+```bash
+export PATH=$SYCL_HOME/build/bin:$PATH
+export LD_LIBRARY_PATH=$SYCL_HOME/build/lib:$LD_LIBRARY_PATH
+```
+
+**Windows**
+```bat
+set PATH=%SYCL_HOME%\build\bin;%PATH%
+set LIB=%SYCL_HOME%\build\lib;%LIB%
+```
+
+and run following command.
+
+```bash
+clang++ -fsycl simple-sycl-app.cpp -o simple-sycl-app.exe -lOpenCL
+```
 
 This `simple-sycl-app.exe` application doesn't specify SYCL device for execution,
 so SYCL runtime will first try to execute on OpenCL GPU device first, if OpenCL
 GPU device is not found, it will try to run OpenCL CPU device; and if OpenCL
```

Comment:
As far as I remember it will try accelerator if CPU device is not found.

Reply:
I changed wording in this section to avoid description of SYCL implementation details.